datafusion/doc/src/udaf.rs

```diff
@@ -0,0 +1,47 @@
+// Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Moving the doc information from to the `datafusion-doc` module makes a lot of sense to me